doc/source/io.rst

```diff
@@ -4633,6 +4635,46 @@ Read only certain columns of a parquet file.
    os.remove('example_pa.parquet')
    os.remove('example_fp.parquet')
 
+
+Handling Indexes
+''''''''''''''''
+
+Dumping a ``DataFrame`` to parquet may include the implicit index as one or more
+columns in the output file. Thus, this code:
+
+.. ipython:: python
+
+    import pandas
+
+    df = pandas.DataFrame({'a': [1, 2], 'b': [3, 4]})
+    df.to_parquet('test.parquet', engine='pyarrow')
+
+creates a parquet file with *three* columns if you use ``pyarrow`` for serialization:
+``a``, ``b``, and ``__index_level_0__``. If you're using ``fastparquet``, the
+index `may or may not <https://fastparquet.readthedocs.io/en/latest/api.html#fastparquet.write>`_
+be written to the file.
+
+This unexpected extra column causes some databases like Amazon Redshift to reject
+the file, because that column doesn't exist in the target table.
+
+If you want to omit a dataframe's indexes when writing, pass ``index=False`` to
+:func:`~pandas.DataFrame.to_parquet`:
+
+.. ipython:: python
+
+    import pandas
+
+    df = pandas.DataFrame({'a': [1, 2], 'b': [3, 4]})
```

Comment:
also no need to recreate the dataframe. The state is preserved between the different code snippets, so the one of the previous code block can be reused.

Reply:
Oh wow, I didn't realize this wasn't _just_ code snippets!